src/main/resources/morph-hbz01-to-lobid.xml

```diff
@@ -976,7 +1076,7 @@
 			value="${itemId}"> <data source="@itemId" name="itemId"> <regexp match="(.*)" format="${1}"/> 
 			</data> </combine> -->
 		<combine name="@aboutSubject" value="${a}">
-			<data source="002[ab]1.a|003-1.a"/>
+			<data source="002[ab]1.a|003-1.a|070-1.a|070a1.a|070b1.a"/>
 			<data source="@id" name="a">
```

Comment:
I reduced this to `<data source="002[ab]1.a"/>` and it doesn't make a difference. So this can be eliminated too.